bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx

```diff
@@ -58,7 +58,7 @@ export default class DropdownListItem extends Component {
         {
           description ?
           (<span id={this.descID} key="describedby" hidden>{description}</span>)
-          : null
+          : (<span id={this.descID} key="describedby" hidden></span>)
```

Comment:
Won't it display "undefined" as a string in the case when description doesn't exist?
Probably it should be:
`<span id={this.descID} key="describedby" hidden>{description ? description : null}</span>`